src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -485,17 +486,19 @@ public async Task<string> AdjustSeatsAsync(Guid organizationId, int seatAdjustme
                         //  being applied forward to the next month's invoice
                         ProrationBehavior = "none",
                         CollectionMethod = collectionMethod,
+                        DaysUntilDue = daysUntilDue,
                     });
                     throw;
                 }
             }
 
-            // Change back the subscription collection method
-            if (collectionMethod != "send_invoice")
+            // Change back the subscription collection method and/or days until due
+            if (collectionMethod != "send_invoice" || daysUntilDue == null)
```

Comment:
Clay just answered this question for me too in Slack 😁

Reply:
Yes.

Stripe's `daysUntilDue == null` option is only valid if `collectionMethod == 'send_invoice'`. Think of it as an optional parameter required only for that collection method. It actually shouldn't be necessary to check daysUntilDue here, I just did out of an abundance of caution in restoring subscription values in case there's something I don't understand about Stripe's internals or they change. 

The logic is that we only change daysUntilDue if it's null, so if it's null we need to update it back to null.